doc/src/devdocs/valgrind.md

```diff
@@ -73,7 +73,7 @@ when julia tries to detect whether it is running under rr:
 --xxxxxx-- WARNING: unhandled amd64-linux syscall: 1008
 --xxxxxx-- You may be able to write your own handler.
 --xxxxxx-- Read the file README_MISSING_SYSCALL_OR_IOCTL.
---xxxxxx-- Nevertheless we consider this a bug.  Please report
+--xxxxxx-- Nevertheless, we consider this a bug.  Please report
```

Comment:
I believe this is the verbatim output so should not be changed (not that it really matters but):
```suggestion
--xxxxxx-- Nevertheless we consider this a bug.  Please report
```